src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs

```diff
@@ -91,6 +91,27 @@ protected override void Up(MigrationBuilder builder)
         {
           table.PrimaryKey($"PK_{DbUnsentEmail.TableName}", x => x.Id);
         });
+
+      builder.CreateTable(
+        name: DbModuleSetting.TableName,
+        columns: table => new
+        {
+          Id = table.Column<Guid>(nullable: false),
+          Host = table.Column<string>(nullable: false),
+          Post = table.Column<int>(nullable: false),
+          EnableSsl = table.Column<bool>(nullable: false),
+          Email = table.Column<string>(nullable: false),
+          Password = table.Column<string>(nullable: false),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey($"PK_DbModuleSetting.TableName", x => x.Id);
```

Comment:
are you sure what your PK has the correct name?

Reply:
fixed